### ISSUE TITLE
Question about the exp setting.

### ISSUE THREAD

**Issue by RunsenXu** [CurryYuan/ZSVG3D#2]
Hi,

Thanks for your interesting work! I want to ask for Nr3D performance of Table 3, do you use class labels of the ground-truth boxes?

Best,
Runsen

**Comment by CurryYuan** [comment]
Hi @RunsenXu ,

We use pre-trained point cloud classifiers as shown in Table 8.

**Comment by RunsenXu** [comment]
Hi,

Thanks for your reply. What are the class labels of the classifiers? NYU40?

Best,
Runsen

**Comment by CurryYuan** [comment]
Hi @RunsenXu ,

We use ScanNet's original class labels.

**Comment by RunsenXu** [comment]
Hi,

Do you mind providing the predicted class label results? Do you use the same cls results as in BUTD-DETR? https://github.com/nickgkan/butd_detr/blob/main/data/cls_results.json

Best,
Runsen

**Comment by CurryYuan** [comment]
Hi @RunsenXu ,

The pre-trained point cloud classifier is from [ULIP2](https://github.com/salesforce/ULIP) , we finetune it with ScanNet labels. The interaction with 2D model is on the fly.

**Comment by RunsenXu** [comment]
Hi, thanks for your response.

Did you use the ScanNet 18 class?

**Comment by CurryYuan** [comment]
We use ScanNet 200 for better results.
